### PR TITLE
Add audiotracks UI into controlbar [Delivers #95256388 #87139744]

### PR DESCRIPTION
**Pull request by chazmatazz**

### CODE REVIEW

File: src/js/view/components/listmenu.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+define([
```
Comment (robwalch) [inline src/js/view/components/listmenu.js]: Too much duplication here. Just make non-togglable a setup configuration option in menu.js.

Comment (robwalch) [inline src/js/view/components/listmenu.js]: Ex:
``` javascript
setup : function (list, selectedIndex, options) {
    ///
    if (list.length > 2 || (options && options.toggle === false)) {
        ///
    } else if (list.length === 2) {
        ///
    }
```

File: src/js/view/controlbar.js [hunk 2]
```diff
@@ -216,12 +222,26 @@ define([
...
+            this.elements.audiotracks.setup();
+
             this._model.mediaModel.on('change:levels', function(model, levels) {
                 this.elements.hd.setup(levels, model.get('currentLevel'));
             }, this);
             this._model.mediaModel.on('change:currentLevel', function(model, level) {
                 this.elements.hd.selectItem(level);
             }, this);
+            this._model.mediaModel.on('change:audioTracks', function(model, audioTracks) {
+                var list = [];
```
Comment (donato) [inline src/js/view/controlbar.js]: var list = _.map(audioTracks, function(track) { return { label : track.name }; })

**Comment by donato** [comment]
This looks pretty good! Just need to fix the PR

**Pull request merged**